src/lamplib/tasks/download.py

```diff
@@ -58,13 +58,11 @@ def fetch_and_install(self):
                         "internal volume your Mac. Verify this assumption with `diskutil list`, substituting the "
                         "appropriate volume name if required. \n\n"
 
-                        "echo 'data' | sudo tee -a /etc/synthetic.conf\n"
+                        sudo mkdir /opt/data
+                        sudo chown $USER /opt/data
+                        echo -e 'data\topt/data' | sudo tee -a /etc/synthetic.conf
```

Comment:
SO doesn't do a great job at showing the link for an answer 😬  I had intended to point to the one with 44 votes, not the one after it. lol

https://apple.stackexchange.com/questions/371908/how-to-make-root-volume-writeable-again-in-catalina/372500#372500